### ISSUE TITLE
Bug report - Bithdays on 29th of February create NaN objects

### ISSUE THREAD

**Issue by phoenix-pnp** [klein0r/ioBroker.birthdays#73]
### The problem

As visible from following screenshots birthdays on 29th of February are not recognized by the adapter.
Correct behaviour would be to move these birthdays to the first of March (Apple and MS move them back to 28th of February, maybe that is the way in the US?)

![image](https://user-images.githubusercontent.com/18349726/147928698-e10d6dde-5e73-4faa-9d2b-bc2b700c4149.png)

### Version of nodejs

14.18.2

### Version of ioBroker js-controller

3.3.22

### Version of Adapter

0.1.7

### Operating system running ioBroker

Raspberry OS "Bullseye"

### Checklist of files to include below

- [X] iobroker.current.log (Please share the logs in debug mode)
- [ ] Contents of the JavaScript browser console (always include in cases of issues with the user interface)

### Additional information & file uploads

**Snippet from debug log:**
```
2022-01-03 13:09:38.884 - debug: birthdays.0 (14139) birthdays: [{"name":"Person 1","birthYear":1940,"dateFormat":"NaN.NaN.NaN","age":null,"daysLeft":null,"_nextBirthday":null},{"name":"Person 2","birthYear":2008,"dateFormat":"NaN.NaN.NaN","age":null,"daysLeft":null,"_nextBirthday":null}]
2022-01-03 13:09:39.023 - debug: birthdays.0 (14139) birthday added: month.NaN.person1
2022-01-03 13:09:39.062 - debug: birthdays.0 (14139) birthday added: month.NaN.person2
2022-01-03 13:09:42.950 - debug: birthdays.0 (14139) filling next with 15 days left
2022-01-03 13:09:42.985 - debug: birthdays.0 (14139) filling nextAfter with 28 days left
```

**Comment by phoenix-pnp** [comment]
As mentioned in #74 I am happy to provide you with a full log thru email or other private channels.
And what I forgot: Thanks for your great work with the adapters and for the trainings! ;)

**Comment by klein0r** [comment]
Fixed in 0.1.8
